src/gui/symbology-ng/qgssymbollayerv2widget.cpp

```diff
@@ -2544,6 +2551,12 @@ void QgsFontMarkerSymbolLayerV2Widget::setSymbolLayer( QgsSymbolLayerV2* layer )
   btnColor->blockSignals( true );
   btnColor->setColor( mLayer->color() );
   btnColor->blockSignals( false );
+  btnBorderColor->blockSignals( true );
```

Comment:
whileBlocking( btnBorderColor )->setColor( mLayer->outlineColor() );

(I realise there's a bunch of offending code around this, but may as well start somewhere!)